### PR TITLE
Many small break-fixes

### PR DESCRIPTION
**Pull request by coderwilson**
Mostly in Nemesis, some in other areas.

### CODE REVIEW

File: nemesis/arena_battles.py [hunk 1]
```diff
@@ -207,15 +207,20 @@ def yojimbo_battle():
...
-    battle.overdrive.yojimbo()  # Backup plan
-
-    memory.main.wait_frames(90)
+    if not Yuna.is_turn():
+        while memory.main.battle_active() and not Yuna.is_turn():
+            if memory.main.turn_ready():
+                CurrentPlayer().defend()
+    if memory.main.battle_active():
+        logger.debug("Yuna Overdrive to summon Yojimbo")
+        Yuna.overdrive(aeon_num=5)
```
Comment (jlward) [inline nemesis/arena_battles.py:216]: You should be able to pass in `Yojimbo` as the argument and it should just work.

File: area/sin.py [hunk 2]
```diff
@@ -133,12 +133,12 @@ def facing_sin():
 
 def inside_sin():
     logger.info("Moving to position next to save sphere")
-    while not pathing.set_movement([247, -237]):
-        if memory.main.diag_skip_possible():
-            xbox.tap_b()
-        elif memory.main.menu_open():
-            xbox.tap_b()
-    logger.info("Moving to next map")
+    #while not pathing.set_movement([247, -237]):
```
Comment (jlward) [inline area/sin.py:136]: We sure we don't need this? Is something else doing this now?

File: nemesis/arena_battles.py (hunk 1 above)
Comment (coderwilson) [reply nemesis/arena_battles.py:216]: Will add this as a future enhancement.

File: area/sin.py (hunk 2 above)
Comment (coderwilson) [reply area/sin.py:136]: Tested and validated before the PR was put up. This was caused due to over-testing any% CSR with under-testing other categories that lead into this point of the game.

**Pull request merged**